workflows/kfdrc_tumor_only_dna_wf.cwl

```diff
@@ -92,247 +95,217 @@ requirements:
-      recommended. Tool will prefilter for germline and pass if expression given",
-    "sbg:suggestedValue": {class: File, path: 6509b6a37f6417197fc158fd, name: dbSNP_v153_ucsc-compatible.converted.vt.decomp.norm.common_snps.vcf.gz}}
-  b_allele_index: {type: 'File?', doc: "Tabix index for b_allele", "sbg:suggestedValue": {
-      class: File, path: 6509b6a37f6417197fc158fc, name: dbSNP_v153_ucsc-compatible.converted.vt.decomp.norm.common_snps.vcf.gz.tbi}}
-  cfree_coeff_var: {type: 'float?', default: 0.05, doc: "Coefficient of variation
-      to set window size. Default 0.05 recommended"}
-  cfree_sex: {type: ['null', {type: enum, name: sex, symbols: ["XX", "XY"]}], doc: "If
-      known, XX for female, XY for male"}
-  hg38_strelka_bed: {type: 'File', doc: "Bgzipped interval bed file. Recommned padding
-      100bp for WXS; Recommend canonical chromosomes for WGS", "sbg:suggestedValue": {
-      class: File, path: 5f500135e4b0370371c051ae, name: hg38_strelka.bed.gz}}
-  hg38_strelka_tbi: {type: 'File?', doc: "Tabix index for hg38_strelka_bed", "sbg:suggestedValue": {
-      class: File, path: 5f500135e4b0370371c051aa, name: hg38_strelka.bed.gz.tbi}}
-  annotsv_annotations_dir_tgz: {type: 'File?', doc: "TAR.GZ'd Directory containing
-      annotations for AnnotSV", "sbg:fileTypes": "TAR, TAR.GZ, TGZ", "sbg:suggestedValue": {
-      class: File, path: 6328ab26d01163633dabcc2e, name: annotsv_311_plus_ens105_annotations_dir.tgz}}
-  unpadded_capture_regions: {type: 'File?', doc: "Capture regions with NO padding
-      for cnv calling"}
-  vep_cache: {type: 'File', doc: "tar gzipped cache from ensembl/local converted cache",
-    "sbg:suggestedValue": {class: File, path: 6332f8e47535110eb79c794f, name: homo_sapiens_merged_vep_105_indexed_GRCh38.tar.gz}}
-  vep_ram: {type: 'int?', default: 32, doc: "In GB, may need to increase this value
-      depending on the size/complexity of input"}
-  vep_cores: {type: 'int?', default: 16, doc: "Number of cores to use. May need to
-      increase for really large inputs"}
-  vep_buffer_size: {type: 'int?', default: 1000, doc: "Increase or decrease to balance
-      speed and memory usage"}
-  dbnsfp: {type: 'File?', secondaryFiles: [.tbi, ^.readme.txt], doc: "VEP-formatted
-      plugin file, index, and readme file containing dbNSFP annotations"}
-  dbnsfp_fields: {type: 'string?', doc: "csv string with desired fields to annotate.
-      Use ALL to grab all"}
-  merged: {type: 'boolean?', doc: "Set to true if merged cache used", default: true}
-  cadd_indels: {type: 'File?', secondaryFiles: [.tbi], doc: "VEP-formatted plugin
-      file and index containing CADD indel annotations"}
-  cadd_snvs: {type: 'File?', secondaryFiles: [.tbi], doc: "VEP-formatted plugin file
-      and index containing CADD SNV annotations"}
+  panel_of_normals: {type: 'File?', secondaryFiles: ['.tbi'], doc: "VCF file (and index) of sites observed in normal. A panel of normals
```

Comment:
Yeah, I guess setting 1000G PON is safest, and we'll have to tell Ops to use GDC and pull it from...I forget, because we can't put it in the public bucket, and probably nor the public project

Reply:
Yeah idk where they'll want it. I'm sure there's something they can set up.